diesel/src/pg/mod.rs

```diff
@@ -6,7 +6,6 @@
 
 pub mod expression;
 pub mod types;
-pub mod upsert;
```

Comment:
This should also have a `#[doc(hidden)]` attribute to prevent it from showing up in the docs. Also there is the enabled-by-default feature `with-deprecated` for enabling deprecated things. So as the name suggests this reexport should only be included if that feature is enabled. That means just add `#[cfg(feature = "with-deprecated")]` to it.

Reply:
Fixed.